static/js/apps/nl_interface/debug_info.tsx

```diff
@@ -205,6 +206,16 @@ export function DebugInfo(props: DebugInfoProps): JSX.Element {
               Overview classification: {debugInfo.overviewClassification}
             </Col>
           </Row>
+          <Row>
+            <b>Query Detection Debug Logs:</b>
+          </Row>
+          <Row>
+            <Col>
+              <pre>
+                {JSON.stringify(debugInfo.queryDetectionDebugLogs, null, 2)}
+              </pre>
+            </Col>
+          </Row>
```

Comment:
nit: variables are arguably a bit more important, and these are more for debugging when things go bad, so move it to after vars?

Reply:
Done.